packages/fxa-auth-server/lib/routes/index.js

```diff
@@ -224,7 +224,7 @@ module.exports = function (
   );
 
   const { cloudTaskRoutes } = require('./cloud-tasks');
-  const cloudTasks = cloudTaskRoutes(log, config);
+  const cloudTasks = cloudTaskRoutes(log, config, statsd);
```

Comment:
Stinks that statsd is passed in vs DI. We have been using it in alot more places.